modules/token-holders/pages/seed-round.adoc

```diff
@@ -0,0 +1,232 @@
++
+....
+g3jww-sbmtm-gxsag-4mecu-72yc4-kef5v-euixq-og2kd-sav2v-p2sb3-pae
+....
++
+This principal represents the beneficiary for which you will create neurons and is used to call the genesis token canister (GTC) on the {IC}.
+. Generate and store the public key for your principal and 12-word seed phrase in a `PUBLIC_KEY` environment variable by running a command similar to the following:
++
+....
+PUBLIC_KEY="$(keysmith public-key -f my-seed-phrase.txt)"
+....
+
+== Create neurons
+
+The next step is to sign and send a message to the genesis token canister (GTC) to create your neuron stake.
+You can do this in one step using a single `dfx canister call` command, or, for added security, as two separate steps that your complete at different times or using different devices.
+For example, you might want to create and sign the message to create neurons on one computer, then copy the message to a different computer or convert the message to a QR code before sending it.
+
+Note the you must have a `dfx.json` file available in the directory from which you are running `dfx` commands that connect to the {IC}. 
+You can create an empty `dfx.json` file in your current directory by running the following command:
+
+....
+touch dfx.json
+....
+
+After creating the `dfx.json` file, you can continue to either the <<Single call>> or <<Separate sign and send steps>> instructions.
+
+=== Single call
+
+To create your neuron stake using a single call:
+
+. Verify that you can access the {IC} by running the following command:
++
+....
+dfx ping ic
+....
+. Call the genesis token canister *canister identifier* and the `create_neurons` method by running a command similar to the following:
++
+....
+dfx canister --network=ic --no-wallet call rno2w-sqaaa-aaaaa-aaacq-cai create_neurons '(\"$PUBLIC_KEY\")'
+....
+
+=== Separate sign and send steps
+
+In some cases, however, you might want to separate the request for creating neurons in separate signing and sending tasks.
+For example, you might want to sign the request but delay sending it for some period of time.
+
+To create your neuron stake using separate sign and send commands:
+
+. Verify that you can access the {IC} by running the following command:
++
+....
+dfx ping ic
+....
+. Create a signed `message.json` file using the principal associated with the identity you created from the PEM file by running a command similar to the following:
++
+....
+dfx canister --network=ic --no-wallet sign --expire-after=1h rno2w-sqaaa-aaaaa-aaacq-cai create_neurons ‘(“PUBLIC_KEY”)’
+....
++
+This command illustrates how to creates a `message.json` to create neurons that is signed using your principal identifier as the message sender and with an expiration window of one hour.
++
+The `--expire-after` option enables you to specify a maximum period of time for the signed message to be valid. The default is 5 minutes (5m).
```

Comment:
The valid time interval to send the message will always be a 5 minutes window ending in the time pint specified by  `--expire-after`. For example, if set `--expire-after=1h` when sign, the generated message can only be sent during [55m, 60m]. The user have to wait for at least 55 minutes.